go.mod

```diff
@@ -1,75 +1,72 @@
 module volcano.sh/apis
 
-go 1.23.0
+go 1.24.3
```

Comment:
Why not 1.24.0

Reply:
since it is / was the latest stable version at the time i opened this pr. in the other pr, https://github.com/volcano-sh/volcano/pull/4342/files, go version was 1.23.7, so i also used the latest patch version as well

Reply:
Generally, the go version 1.xx.0 may have some unstable factors. Is it better to use 1.24.3?

Reply:
I think 1.24.0 is ok, as when others import this API repo, they also need build with go v1.24.3, we should maintain a minimum satisfactory go version.